__tests__/Login.test.js

```diff
@@ -1,59 +1,54 @@
-/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+/** This Source Code Form is subject to the terms of the Mozilla Public License,
  * v. 2.0. If a copy of the MPL was not distributed with this file, You can
  * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
  * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
  *
  * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
  * graphic logo is a trademark of OpenMRS Inc.
  */
-import React, { Component } from 'react';
-import { mount } from 'enzyme';
-import Enzyme from 'enzyme';
-import Adapter from 'enzyme-adapter-react-15.4';
-
-import Login from '../app/js/components/Login';
-import LoginForm from '../app/js/components/presentational/LoginForm'
+import React, { Component } from "react";
+import { mount } from "enzyme";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-15.4";
```

Comment:
Can you configure enzyme and adapter in your setup.js file instead of having to import it in every test file.

Reply:
Good observation, I have added them to the `setUpTests.js` file at the root of the application.